src/lang/eval/EvalUtil.ml

```diff
@@ -40,10 +40,10 @@ module EvalGas = ScillaGas (SR) (ER)
 open EvalSyntax
     
 (* Return a builtin_op wrapped in EvalMonad *)
-let builtin_executor i arg_tps arg_lits =
+let builtin_executor f arg_tps arg_lits loc =
```

Comment:
Again here, once you introduce `builtin_ident`, you can extract the location from `f`, so you won't need the `loc` parameter.